nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1059,14 +1059,352 @@ def get_execution_order(cls):
+
+        - meta-training training: Train the inner loop learner (i.e., slow parameters)
+        for a specific task
+        - meta-training testing: Train the outer loop learner (i.e., fast parameters)
+        by minimizing the test loss
+
+        - meta-testing training: Train the inner loop learner continually on a sequence
+        of holdout tasks
+        - meta-testing testing: Evaluate the inner loop learner on the same tasks
+
+    The parameters for a model used in a meta-continual learning setup are broken down
+    into 2 groups:
+
+        - fast parameters: Used to update slow parameters via online learning during
+        meta-training training, and updated along with slow parameters during
+        meta-training testing
+        - slow parameters: Updated during the outer loop (meta-training testing)
+    """
+
+    def setup_experiment(self, config):
+        """
+        Configure the experiment for training
+
+        :param config: Dictionary containing the configuration parameters, most of
+        which are defined in SupervisedExperiment, but some of which are specific to
+        MetaContinualLearningExperiment
+
+            - experiment_class: Class used to run experiments, specify
+            `MetaContinualLearningExperiment` for meta-continual learning
+            - adaptation_lr: Learning rate used to update the fast parameters during
+            the inner loop of the meta-training phase
+            - tasks_per_epoch: Number of different classes used for training during the
+            execution of the inner loop
+            - slow_batch_size: Number of examples in a single batch used to update the
+            slow parameters during the meta-training testing phase
+            - replay_batch_size: Number of examples in a single batch sampled from all
+            data, also used to train the slow parameters
+        """
+
+        # Simplify loading
+        config["model_args"]["num_classes"] = config.get("num_classes", )
+
+        super().setup_experiment(config)
+        self.train_slow_loader = self.train_loader
+        self.train_fast_loader, self.val_fast_loader, self.train_replay_loader = \
+            self.create_fast_slow_loaders(config)
+
+        self.epochs_to_validate = []
+        self.tasks_per_epoch = config.get("tasks_per_epoch", 1)
+        self.num_classes = min(
+            config.get("num_classes", 50),
+            self.train_fast_loader.sampler.num_classes
+        )
+
+        self.adaptation_lr = config.get("adaptation_lr", 0.03)
+
+        self.batch_size = config.get("batch_size", 5)
+        self.val_batch_size = config.get("val_batch_size", 15)
+        self.slow_batch_size = config.get("slow_batch_size", 64)
+        self.replay_batch_size = config.get("replay_batch_size", 64)
+
+    @classmethod
+    def create_loaders(cls, config):
+        """Create train and val dataloaders."""
+
+        # TODO this method needs to (a) be a class method, and (b) return 2 dataloaders
```

Comment:
Is this `TODO` still relevant?

Reply:
It's not, since I already addressed it. I will remove this line.